### PR TITLE
Add new blog post for GSoC - User Interface

### PR DESCRIPTION
**Pull request by davidemdot**
I am preparing the **animated GIF** for adding it in the post, so I will update the MD file soon.

---
title: "Google Summer of Code 2019: User Interface"
id: gsoc-2019-user-interface
---

### CODE REVIEW

**Comment by davidemdot** [comment]
User Interface

**Comment by davidemdot** [comment]
Thanks, @Siedlerchr! I have done those changes.

File: _posts/2019-06-11-GSoC - UserInterface.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+id: gsoc-2019-user-interface
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+bg: jabref-font
+color: white
+---
+
+Hi!
+We keep working on the [LaTeX Integration project for the Google Summer of Code](https://summerofcode.withgoogle.com/projects/#6055042405105664).
+
+As we told last week, any feedback is more than welcome!
```
Comment (tobiasdiez) [inline _posts/2019-06-11-GSoC - UserInterface.md]: "As we told last week" sounds a bit odd (remove it?)

**Comment by davidemdot** [comment]
Sure! It is done :D

**Pull request merged**